### PR TITLE
UPSTREAM: <carry>: bump tag version

### PR DESCRIPTION
**Pull request by soltysh**
/assign @wking @vrutkovs

### CODE REVIEW

**Comment by vrutkovs** [comment]
LGTM

File: REBASE.openshift.md [hunk 1]
```diff
@@ -80,6 +80,27 @@ Before incorporating upstream changes you may want to:
...
   resolve it automatically (https://git-scm.com/book/en/v2/Git-Tools-Rerere)
 
+## Send email annoucing you're starting work
+
+To better spread the information send the following email:
+
+```
+Title: k8s <version> bump is starting...
+
+I'm staring the process of updating our fork to bring in
```
Comment (sttts) [inline REBASE.openshift.md:90]: starting

**Comment by vrutkovs** [comment]
e2e-gcp is flaking due to https://github.com/openshift/release/pull/17598

**Pull request merged**